frontend/src/features/idir/search/table/CompanySearchResultColumnDef.tsx

```diff
@@ -51,7 +47,7 @@ export const CompanySearchResultColumnDef: MRT_ColumnDef<CompanyProfile>[] = [
         <>
           {mailingAddress?.addressLine1}<br />
           {country[0]?.name}<br />
```

Comment:
Should we use the same check here as for the province below?

Reply:
as in something like `getDefaultRequiredVal("", country?.[0]?.name)` or something equivalent

Reply:
yeah - good idea to use that helper function.